lib/order.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'menu' 
```

Comment:
Do you need to require menu here if you're using dependency injection on line 5?